scripts/integrations.json

```diff
@@ -39,6 +42,13 @@
     "@astropub/icons",
     "astro-json-element"
   ],
+  "overlay": [
+    "@astrojs/tailwind",
+    "@astrojs/sitemap",
+    "@astrojs/mdx",
+    "astro-compress",
```

Comment:
This should be removed as soon as Spotlight is available

Reply:
what is this Spotlight magic you speak of?